automation/rancher-partner-charts/app-readme.md

```
@@ -14,7 +14,7 @@ Common use cases include:
 - Customer identity and access management (CIAM)   
 - Identity federation      
 
-### Free Open Source Software 
+### Free Open Source Software
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)